template2/build.ps1

```diff
@@ -0,0 +1,97 @@
+# KoreBuild 2.0
```

Comment:
The old script used to set `REPO_FOLDER`. I think there are some builds that depend on it